doc/api/util.md

```diff
@@ -14,6 +14,60 @@ it:
 const util = require('node:util');
 ```
 
+## `util.addSafeAbortSignalAbortListener(signal, resource)`
```

Comment:
Bikeshed: can we get a shorter name? `util.addAbortListener`

Reply:
Changed to your suggestion